packages/extension-vscode/README.md

```diff
@@ -1,16 +1,21 @@
-# webhint extension for Visual Studio Code
+# VS Code
 
 Use `webhint` to improve your website - during development.
 
 This extension runs and reports diagnostics for workspace files
 based on `webhint` analysis.
 
+![VS Code running the webhint extension][vscode gif]
+
+The extension is still in beta, please check the
+[troubleshooting section][troubleshoot] and if you cannot find an
+answer [open an issue in GitHub][issue github].
+
 ## Prerequisites
```

Comment:
I think we should drop this section entirely since the extension will prompt the user to install webhint if it's not available. Agreed?

Reply:
SGTM

Reply:
I've updated the PR